src/zepben/evolve/streaming/grpc/connect.py

```diff
@@ -50,28 +51,66 @@ def _grpc_channel_builder_from_password(client_id: str, username: str, password:
 
 
 def connect_tls(host: str = "localhost", rpc_port: int = 50051) -> grpc.aio.Channel:
```

Comment:
did you have that code that allowed you to pass in a ca as well? i know we didn't get it working but fairly sure it was mostly correct so happy for you to add it here.

Reply:
Yes, I have the code change in my local. Sure, I will added that part in the next commit.